### PR TITLE
Remove avri documentation

### PR DESCRIPTION
**Pull request by timvancann**
## Proposed change

This removes the documentation for the deprecated component Avri 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/44478
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
This removal is incomplete. Please make sure all links to this page are cleaned up as well.

**Comment by w1ll1am23** [comment]
@frenck seems like this could result in confusion for users if the documentation is completely removed? Should something be left behind in order for users who stumble across old forum posts or something to indicate it was removed in release X?

I don't think this has been done previously, but seems like it could help people out.

**Comment by frenck** [comment]
If you have a suggestion for that, sure, feel free to create a PR to provide such a framework or start a discussion somewhere. But let's keep it out of this PR, as it is out of scope for current running things.

File: source/_posts/2020-03-18-release-107.markdown [hunk 1]
```diff
@@ -139,7 +139,6 @@ Screen recording of the new card picker and stack editor.
 ## New Integrations
 
 - Add platform Ezviz ([@BaQs] - [#30378]) ([ezviz docs]) (new-integration)
-- Add Avri waste collection sensor ([@timvancann] - [#31134]) ([avri docs]) (new-integration)
```
Comment (frenck) [inline source/_posts/2020-03-18-release-107.markdown:142]: These lines should not have been removed, as they are still true. Only the link to the documentation should have been removed.

(this applies to all other occurrences as well).

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by MartinHjelmare** [comment]
Please update the brands repo too.

**Comment by frenck** [comment]
Added a brands PR to clean it up

**Review by frenck — APPROVED** [review]
Thanks! 👍

**Pull request merged**